pose_attacker_node:
  ros__parameters:
    'use_sim_time': true,
    'offset_x': 0.0,
    'offset_y': 0.0,
    'offset_z': 0.0,
    'offset_yaw_deg': 90.0,
    'rate_hz': 34.0,
    'offset_drift_x': 0.9,
    'offset_drift_y': 0.0,
    'offset_drift_yaw_deg': 0.0